source/ocsp-support/tests/README.rst

```diff
@@ -208,6 +208,14 @@ Drivers may wish to use a smaller value for ``serverSelectionTimeoutMS`` to
 speed up tests (otherwise server selection will spin for the entire
 duration even after a driver encounters a TLS error early).
 
+Testing on Linux
+----------------
+
+Drivers MUST test on Linux platforms that have server support for OCSP stapling.
```

Comment:
Should we suggest what linux platforms drivers should test against? Go currently tests against RHEL 7.0 (+ MacOS and Windows).

Reply:
Good point. I added a note to consider using Ubuntu 20.04 (C uses `ubuntu2004-small`) or RHEL 7.